src/js/tabs/firmware_flasher.js

```diff
@@ -409,6 +420,13 @@ firmware_flasher.initialize = function (callback) {
         $('select[name="options"]').select2({ closeOnSelect: false });
         $('select[name="commits"]').select2({ tags: true });
 
+        $('select[name="radioProtocols"]').on("select2:select", function() {
```

Comment:
@limonspb can you check this doesn't contradict the setting of this value based on the build key ? i.e. its ok here, so long as the application of the build key query overrides it.

Reply:
@blckmn good note.
I have updated PR. Now if build key exists (length === 32) then it ignores protocol from localStorage.